pos_order_to_sale_order/README.rst

```diff
@@ -0,0 +1,130 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+=======================
+POS Order To Sale Order
+=======================
+
+
+This module extends the functionality of point of sale to allow you create
+Sale Orders from the Point of sale.
+
+It adds a new button, below the current bill on the point of sale front end UI,
```

Comment:
In the POS UI, a button has been added to create the sales order and discard the current POS order